doc/getstarted/README.md

```diff
@@ -0,0 +1,110 @@
+# Get started 
+
+It's recommended to begin by exploring the 👉 **[basic terms](../design/architecture#1-basic-terms)** 👈 to better understand the documentation.
+
+## Simple deploy
+
+```
+docker run --network host emcmongoose/mongoose-base
+```
+
+☝️ This command will launch mongoose in **standalone mode**, and the **storage mock** will be used as the tested [storage](../design/architecture#1-basic-terms).
+
+> 📘 More about:
+> * **[Modes](../design/modes)**
+> * **[Storage mock/drivers](../design/architecture#21-storage-driver)**
+> * **[Other ways to deploy](../deployment)**
+
+## Simple load tuning
+
+```
+docker run --network host emcmongoose/mongoose-base \
+    --load-op-limit-count=10000 \
+    --item-data-size=1KB
+```
+
+☝️ This command will **"create"** 10000 "[items](../design/architecture#1-basic-terms)" on storage (in this case not real storage, but storage mock) and each "item" will have size "1KB".
+
+```
+docker run --network host emcmongoose/mongoose-base \
+    --read \
+    --item-input-path="some path to item on storage" \
+    --load-step-limit-time=60s \
+    --load-op-recycle=true
+```
+
+☝️ This command will perform **"read"** operation for 60 seconds. Mongoose will "read" all items from specified "item-input-path" (for example, for File System (as Storage) path = `/path/to/directory`) and when "items" run out mongoose will "read" them again (`recycle`).
+
+> 📘 More about:
+> * **[Items](../usage/item)**
+> * **[CLI arguments](../usage/input/cli)** and **[configuartion options](../usage/input/configuration)**
+> * **[Operation types](../usage/load/operations/types)**
+
+## Simple scenario
```

Comment:
Introduction is missed. Like: Mongoose also supports input parameters using 'scenarios' - script files defining a workload.

Reply:
maybe this should be included: Any [JSR-223](https://en.wikipedia.org/wiki/Scripting_for_the_Java_Platform) compatible |[XML](https://en.wikipedia.org/wiki/XML)|[ANSI C, Java, .Net, JS]. It's from comparison page. Nor sure though